WatchDogIntelliJPlugin/src/nl/tudelft/watchdog/intellij/logic/InitializationManager.java

```diff
@@ -51,7 +51,7 @@
     private InitializationManager(Project project) {
```

Comment:
This is likely a problem for accessing the local settings + database of legacy users

Reply:
I have created a forum post to ask this: https://intellij-support.jetbrains.com/hc/en-us/community/posts/115000777330-Changing-plugin-ID-with-update